torch/storage.py

```diff
@@ -237,6 +237,17 @@ def _new_shared(cls, size, *, device='cpu'):
     def untyped(self):
         return self
 
+    def byteswap(self, dtype):
+        """Swaps bytes in underlying data"""
+        elem_size = torch._utils._element_size(dtype)
+        # for complex types, don't swap first and second numbers
+        if dtype.is_complex:
+            elem_size = max(int(elem_size / 2), 1)
+        for i in range(int(len(self) / elem_size)):
+            for k in range(int(elem_size / 2)):
+                self[i * elem_size + k], self[i * elem_size + elem_size - k - 1] = \
```

Comment:
Ok, could you add `def __setitem__(self, *args, **kwargs): ...  # noqa: E704`, up next to the similar `def __getitem__` on line 31 above? That should fix the mypy issue

I'm surprised we don't currently have this mypy failure in our existing tests. That's separate from your PR, I'll look into that soon